prerendering.bs

```diff
@@ -259,23 +304,48 @@ Every {{Document}} has an <dfn for="Document">activation start time</dfn>, which
 
      <p class="note">Currently, cross-site prerendering is not specified or implemented, although we have various ideas about how it could work in this repository's explainers.
 
-  1. If |referrerDoc|'s [=Document/prerendering traversables map=][|startingURL|] [=map/exists=], then return.
+  1. [=list/For each=] |record| of |referrerDoc|'s [=Document/prerender records=]:
+      1. If |record|'s [=prerender record/starting URL=] is |startingURL|, then return.
```

Comment:
On the prefetch side, it doesn't look like we spec any deduplication. I suppose it could be implicit from the prefetch steps themselves being optional?

Implementation-wise, currently both prefetch and prerender only dedupe based on the exact URL.

So I suppose the options are:
1. Remove the explicit deduplication from this prerender spec
2. Add explicit deduplication based on exact matches to the prefetch spec
3. Add explicit deduplication based on NVS matches to the prefetch and prerender specs and update the chromium implementation
4. Add explicit deduplication based on exact matches to the prefetch spec and add explicitly optional NVS based deduplication to prefetch and prerender specs

Can we think of a scenario where duplicate requests would be desirable for a UA to allow? If not, option 3 seems appropriate. WDYT?

Reply:
I can imagine a scenario (while the site _can_ adapt to the query param being different, an exact match provides a "better" response which is fully ready faster), but I'm not sure it's worth indexing on. I have a leaning toward 3 as well, but maybe this should fork into an issue in case we want to weigh this a little more (and ask others)?

Reply:
Filed #315